src/controller/java/src/matter/onboardingpayload/OptionalQRCodeInfo.kt

```diff
@@ -37,4 +37,40 @@ class OptionalQRCodeInfoExtension : OptionalQRCodeInfo() {
   var int64: Long = 0
   var uint32: Long = 0
```

Comment:
I know this is not introduced by this PR, the uint32 and uint64 fields are both declared as Long in Kotlin, these are not appropriate , Kotlin use UInt for uint32 and ULong for uint664

Reply:
Makes sense. I'll get that fixed up